demisto_sdk/commands/common/tools.py

```diff
@@ -2125,7 +2129,8 @@ def get_file_displayed_name(file_path):
                        FileType.INDICATOR_FIELD, FileType.LAYOUTS_CONTAINER, FileType.PRE_PROCESS_RULES,
                        FileType.DASHBOARD, FileType.WIDGET,
                        FileType.REPORT, FileType.JOB, FileType.WIZARD]:
-        return get_json(file_path).get('name')
+        res = get_json(file_path)
+        return res.get('name') if isinstance(res, dict) else res[0].get('name')
```

Comment:
if it's a list, is it always of size 1? 

Perhaps add a check: if res is not a dict, and ((not list) or (list of len >1)) log a warning that it's an unexpected value and the reason

Reply:
Let's talk about it